cmd/dynkeepalived/dynkeepalived.go

```diff
@@ -54,7 +54,7 @@ func main() {
 		},
 	}
 	rootCmd.PersistentFlags().StringP("cluster-config", "c", "", "Path to cluster-config ConfigMap to retrieve ControlPlane info")
-	rootCmd.Flags().Duration("check-interval", time.Second*10, "Time between keepalived watch checks")
+	rootCmd.Flags().Duration("check-interval", time.Second*15, "Time between keepalived watch checks")
```

Comment:
This is just reducing the race chance, right? Not addressing the source

Reply:
Yes and Not,
Well, In case bootstrap  failed to read master nodes details from OCP API , we have the following flow:

1. Master-x send a request to the unicast server at bootstrap node 
2. The unicast server at the bootstrap should : 
  2.1 add master-x IP address to an array (stores the clients/masters IPs learned by the server)
  2.2 Reply with his IP address
3. Master-x will add bootstrapIP to its configuration, and after debouncer of 3 ( poll interval ) , the new config will be written to keepalived config file 
4. From the bootstrap side, In the next poll interval the bootstrap will include master-x IP in his configuration ( same here 3*toll interval)

So, in the worst case, master-x will have 2 poll interval (2*10) before it adds the bootstrap IP, and bootstrap has 3 poll intervals (3*5 seconds) 
I guess we can delete this fix,  I wanted to see green CI first